tests/market/test_market.py

```diff
@@ -123,23 +129,29 @@ def test_open_market(vega, page):
         is_amendment=False,
     )
 
-    submit_order(vega, MM_WALLET, market_id, "SIDE_BUY",
+    expect(page.get_by_test_id(liquidity_supplied
+                               ).get_by_test_id(item_value)).to_have_text("0.00 (0.00%)")
+    submit_order(vega, MM_WALLET.name, market_id, "SIDE_BUY",
```

Comment:
Perhaps add some comments here to explain how/what you are doing to the market

Reply:
done